scripts/gen_mmu.py

```diff
@@ -369,6 +1083,8 @@ def parse_args():
 
     parser.add_argument("-i", "--input",
                         help="Input file from which MMU regions are read.")
+    parser.add_argument("-c", "--config",
```

Comment:
I wouldn't do it this way.
All Kconfigs end up as symbols in the ELF file. You can use elftools to read this directly out of zephyr_prebuilt.elf
Have a look at scripts/gen_gdt.py for an example.